cmd/agent.go

```diff
@@ -352,9 +371,7 @@ func (ar *AgentRunner) runInstance() error {
 			return err
 		}
 
-		_, err = runnerInstance.Configure(&proto.ConfigureRequest{
-			Config: pluginConfig.Config,
-		})
```

Comment:
Let's keep sending the full Request here. More explicit for what's actually happening.